src/CDSAPI.jl

```diff
@@ -1,27 +1,32 @@
 module CDSAPI
 
-export py2ju, retrieve
+using HTTP
+using JSON
+using Base64
 
-using HTTP, JSON, Base64
+export
+    py2ju
 
 function retrieve(name, params, filename)
-    cred = Dict()
-    open(joinpath(homedir(), ".cdsapirc"), "r") do f
-        dicttxt = readlines(f)
-        for i in dicttxt
-            tmp = split(i, ":"; limit=2)
-            cred[tmp[1]] = lstrip(tmp[2])
-        end
+    creds = Dict()
+    lines = readlines(open(joinpath(homedir(), ".cdsapirc"), "r"))
+    for line in lines
+        key, val = strip.(split(line, ":"; limit=2))
```

Comment:
Do we really need the `limit=2` option? :) Perhaps erasing it will make the code cleaner.

Reply:
Hi I think yes, because the API key url it is "url: https://cds.climate.copernicus.eu/api/v2" there is second colon in the url :/

Reply:
Oh good point. The option `limit` makes sense :+1: